lib/hammer_cli_foreman/subnet.rb

```diff
@@ -1,5 +1,28 @@
 module HammerCLIForeman
 
+   module SubnetUpdateCreateCommons
+
+    def self.included(base)
+      base.option "--dns", "DNS_NAME", _("DNS Proxy Name to use within this subnet")
```

Comment:
Maybe just "DNS Proxy to use within this subnet" to match the strings in UI. Similarly for the other strings.